src/BloomExe/ToPalaso/BetterTooltipTransparentOverlay.cs

```diff
@@ -4,24 +4,29 @@
 using System.Security.Permissions;
 using System.Text;
 using System.Windows.Forms;
+using System.Drawing;
 
 namespace Bloom.ToPalaso
 {
 	class BetterTooltipTransparentOverlay : ContainerControl
 	{
-		public BetterTooltipTransparentOverlay()
+		private readonly Control _overlayedControl;
+
+		public BetterTooltipTransparentOverlay(Control overlayedControl)
 		{
+			_overlayedControl = overlayedControl;
+
 			// Disable painting the background.
-			this.SetStyle(ControlStyles.Opaque, true);
-			this.UpdateStyles();
+			SetStyle(ControlStyles.Opaque, true);
```

Comment:
It's highly unobvious what this does, even after reading the cryptic description in MSDN. Would it be clearer to just override onPaintBackground to do nothing? Setting the property AllPaintingInWmPaint more obviously suggests the behavior I think you want, though whether it will actually produce that I don't know.